docker/Dockerfile.rpm-yum

```
@@ -71,6 +71,7 @@ RUN make PREFIX=${DIST_DIR} cmds
 
 WORKDIR $DIST_DIR/..
 COPY packaging/rpm .
+COPY deployments/systemd/* ${DIST_DIR}/
```

Is there a reason that we don't use: 
```suggestion
COPY deployments/systemd/ ${DIST_DIR}/
```
Like we do for the other Dockerfiles?

---

done